<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GloSConfigClass</class>
 <widget class="QMainWindow" name="GloSConfigClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GloSConfigurator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="b_Save">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lE_Port">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cB_Audio">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="b_Audio">
    <property name="geometry">
     <rect>
      <x>399</x>
      <y>19</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Audio device</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lE_Audio">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>152</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="b_Port">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cB_Ports"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="GloSConfig.qrc"/>
 </resources>
 <connections/>
</ui>
